### ISSUE TITLE
Search

### ISSUE THREAD

**Issue by damoclark** [emuell/restic-browser#30]
Hi @emuell 

Last idea/suggestion. :)

Search functionality would be really useful for restic-browser.  Initially perhaps on the path and tags, although date/time ranges could be used too.  With the introduction of #29 you could use that same text box with search syntax.

Just sharing these ideas after using restic-browser for a little while now.  But they are only suggestions.  I fully recognise that you are likely sharing your work in case it is useful for someone else, just scratching an itch, or having some fun hacking some code.  In other words, I have no expectations, and simply appreciate an opportunity to provide feedback that may help make a good idea even better.

In any case, thank-you Eduard for sharing your work. :)

Damo.

**Comment by emuell** [comment]
Yes, indeed. With the result of the "recursive ls" referred to [here](https://github.com/emuell/restic-browser/issues/29#issuecomment-1233929647), a search could indeed be easily performed on that as well. 

Thanks for sharing your ideas. Much appreciated!

**Comment by damoclark** [comment]
No problem.  Another approach might be to use the restic-find command, which I think will do all the heavy lifting for you in terms of locating and returning only the specific items, rather than pulling back all the data, and having to filter in Restic-browser.
